lib/friendly_shipping/services/usps_international.rb

```diff
@@ -17,7 +17,7 @@ class UspsInternational
         rectanglular: 'RECTANGULAR',
         roll: 'ROLL',
         variable: 'VARIABLE'
-      }
+      }.freeze
```

Comment:
👍

Reply:
Does the freeze header only affect strings?

Reply:
It works on hashes too. It's supposed to [reduce object allocations](https://www.honeybadger.io/blog/when-to-use-freeze-and-frozen-in-ruby/). I was just doing it here because Rubocop was complaining.